hypothesis-python/src/hypothesis/extra/django/_fields.py

```diff
@@ -36,6 +42,14 @@
     from datetime import tzinfo  # noqa
     from typing import Any, Type, Optional, List, Text, Callable, Union  # noqa
 
+
+def get_tz_strat():
```

Comment:
I think this function might be dead code, added before you split model and form datetimes.  Is it still used anywhere?